tests/bluetooth/bsim_bt/bsim_test_mesh/tests_scripts/provision/pb_adv_oob_auth.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright 2021 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source $(dirname "${BASH_SOURCE[0]}")/../../_mesh_test.sh
+
+for i in {0..10} ; do
+	RunTest mesh_provision_pb_adv_oob_auth \
+		prov_device_pb_adv_oob_auth \
+		prov_provisioner_pb_adv_oob_auth -- -argstest test-step=$i
+done
```

Comment:
Wouldn't it be faster/simpler to perform the loop within the test scenario?

Reply:
There is no way to unprovision device.

Reply:
`bt_mesh_reset()`

Reply:
it doesn't seem to work 
```
d_01: @00:00:00.002648  [00:00:00.002,624] <wrn> bt_ecc: Callback already registered
d_01: @00:00:00.002648  [00:00:00.002,624] <err> bt_mesh_prov: Failed to generate public key (-114)
```
not possible to start mesh again because of public key error